src/uiTest/scala/com/virtuslab/gitmachete/uitest/UIScenarioSuite.scala

```diff
@@ -0,0 +1,103 @@
+package com.virtuslab.gitmachete.uitest
+
+import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite
+import com.virtuslab.gitmachete.testcommon.SetupScripts.SETUP_README_SCENARIOS
+import org.junit._
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.virtuslab.ideprobe.ProbeDriver
+
+@RunWith(classOf[JUnit4])
+class UIScenarioSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_README_SCENARIOS) {
+
+  import UISuite._
+  UISuite.setup()
+
+  private val project = intelliJ.project
```

Comment:
I have a strong suspicion that this the source of failures on CI.

`intelliJ` is a `def`, which points to a var `org.virtuslab.ideprobe.RunningIntelliJPerSuiteBase#runningIntelliJFixture`. This underlying `var` probably changes b/w test cases... while `project` points to the same instance.

Either make `project` a `def`, or un-inline

Reply:
(oh... these comments are actually more _suited_ to `UITestSuite`)